lib/player_factory.rb

```diff
@@ -18,4 +20,14 @@ def create_players(game_mode)
       {X => ComputerPlayer.new(X), O => HumanConsolePlayer.new(O, ui)}
     end
   end
+
+  def create_web_players(game_mode)
```

Comment:
It looks like there is some duplication here. Is there a way to just dynamically choose the right human player?